eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/constants/EventMeshConstants.java

```diff
@@ -31,7 +33,7 @@ public class EventMeshConstants {
 
     public static final String PROTOCOL_GRPC = "grpc";
 
-    public static final String DEFAULT_CHARSET = "UTF-8";
+    public static final String DEFAULT_CHARSET = StandardCharsets.UTF_8;
```

Comment:
public static final String DEFAULT_CHARSET = StandardCharsets.UTF_8.name();
or delete:
![image](https://user-images.githubusercontent.com/91315508/234739723-f4da2db6-231b-48b1-b02b-6e1d872702ce.png)

Then replace it with `Constants.DEFAULT_CHARSET.name()`:

In these classes need to be replaced:
![image](https://user-images.githubusercontent.com/91315508/234740195-f881ae5f-21ca-47e1-90db-4ca8b8cfc80d.png)